juju/juju
# lp:1762700, iterfaces.py requires the real interfaces file

## Description of change

juju cloudinit for containers is failing in a networking script because the network interface file is not found.  To make it found, don't specify a tmp file instead of the real file.  The script does create a tmp itself and a bak copy as well.

## QA steps

juju bootstrap localhost test
juju deploy ubuntu --to lxd

both the machine and container should deploy successfully.

## Documentation changes

N/A

## Bug reference

https://bugs.launchpad.net/juju/+bug/1762700
